sampleCode/CSharp/ConsoleApp/Workflows/CreateProjectWorkflow.cs

```diff
@@ -0,0 +1,93 @@
+﻿namespace ConsoleApp.Workflows;
+
+/// <summary>
+/// A workflow that describes the process to create and fill a Project
+/// </summary>
+public class CreateProjectWorkflow : IWorkflow
+{
+    public static void Examples()
+    {
+        /* The first step is to create the Project itself
+         * Aa unique Title, Aggregation Scheme Id, and HouseholdSetId are all required to define a Project
+         */
+
+        // Get a list of all valid Aggregation Schemes
+        var aggregationSchemes = AggregationSchemes.GetAggregationSchemes();
+        // Choose the one you would like to use
+        int aggregationSchemeId = 8; // The default
+        
+        
+        // Select a HouseholdSetId from among the valid options in the AggregationScheme
+        int householdSetId = 1;
+
+        
+        // Get a list of all valid Data Sets for the Aggregation Scheme
+        var datasets = DataSets.GetDataSets(aggregationSchemeId);
+        // Choose the one you would like to use -- must be compatible with your chosen HouseholdSetId
+        int dataSetId = 96;
+
+        
+        // Now we can define the project with its required properties
+        Project project = new Project
+        {
+            Title = $"PHX-11757 - Test - {Guid.NewGuid()}",
```

Comment:
@TRJ-Implan can we remove the ticket number and test, and replace with "Project Name" then guid?